cartography/intel/aws/securityhub.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+import logging
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_hub(boto3_session: boto3.session.Session) -> Dict:
+    client = boto3_session.client('securityhub')
+    try:
+        return client.describe_hub()
+    except client.exceptions.ResourceNotFoundException:
+        return {}
+
+
+@timeit
+def load_hub(
+    neo4j_session: neo4j.Session,
+    data: Dict,
+    current_aws_account_id: str,
+    aws_update_tag: int,
+) -> None:
+    ingest_hub = """
+    WITH {Hub} AS hub
```

Comment:
Could we use `UNWIND` here?

Reply:
There's only a single one per account/region.